packages/flutter/test/widgets/mouse_region_test.dart

```diff
@@ -639,12 +639,14 @@ void main() {
 
     await tester.pumpWidget(
       Center(
-          child: Container(
-              child: HoverFeedback(
-        key: feedbackKey,
-        onEnter: () => numEntries++,
-        onExit: () => numExits++,
-      ))),
+        child: Container(
+          child: HoverFeedback(
+            key: feedbackKey,
+            onEnter: () => numEntries++,
+            onExit: () => numExits++,
```

Comment:
onEnter and onExit expect `void` return values but this code returns integers.

Prefer:
```dart
  onEnter: () { numEntries += 1; },
  onExit: () { numExits += 1; },
```

Also maybe numEntries -> numEnters or numEntrances. Entries usually is used to mean the number of things in a list.

Reply:
Fixed, thanks.